tools/cbmc/.gitignore

```diff
@@ -2,3 +2,5 @@ cbmc.txt
 property.xml
 coverage.xml
 *.goto
+TAGS-*
+html
```

Comment:
Please check the template starter kit and add these things there if they aren't already there.  (Final new line missing.)